src/pages/office/cat-a-mod1/office.cat-a-mod1.page.ts

```diff
@@ -0,0 +1,607 @@
+        withLatestFrom(currentTest$.pipe(
+          select(getTestSummary),
+          select(getRouteNumber))),
+        map(([outcome, route]) => this.outcomeBehaviourProvider.isVisible(outcome, 'routeNumber', route)),
+      ),
+      displayIndependentDriving$: currentTest$.pipe(
+        select(getTestOutcome),
+        withLatestFrom(currentTest$.pipe(
+          select(getTestSummary),
+          select(getIndependentDriving))),
+        map(([outcome, independent]) =>
+          this.outcomeBehaviourProvider.isVisible(outcome, 'independentDriving', independent)),
+      ),
+      displayCandidateDescription$: currentTest$.pipe(
+        select(getTestOutcome),
+        withLatestFrom(currentTest$.pipe(
+          select(getTestSummary),
+          select(getCandidateDescription))),
+        map(([outcome, candidate]) =>
+          this.outcomeBehaviourProvider.isVisible(outcome, 'candidateDescription', candidate)),
+      ),
+      displayIdentification$: currentTest$.pipe(
+        select(getTestOutcome),
+        withLatestFrom(currentTest$.pipe(
+          select(getTestSummary),
+          select(getIdentification))),
+        map(([outcome, identification]) =>
+          this.outcomeBehaviourProvider.isVisible(outcome, 'identification', identification)),
+      ),
+      displayWeatherConditions$: currentTest$.pipe(
+        select(getTestOutcome),
+        withLatestFrom(currentTest$.pipe(
+          select(getTestSummary),
+          select(getWeatherConditions))),
+        map(([outcome, weather]) =>
+          this.outcomeBehaviourProvider.isVisible(outcome, 'weatherConditions', weather)),
+      ),
+      displayAdditionalInformation$: currentTest$.pipe(
+        select(getTestOutcome),
+        withLatestFrom(currentTest$.pipe(
+          select(getTestSummary),
+          select(getAdditionalInformation))),
+        map(([outcome, additional]) =>
+          this.outcomeBehaviourProvider.isVisible(outcome, 'additionalInformation', additional)),
+      ),
+      displayEco$: currentTest$.pipe(
+        select(getTestOutcome),
+        withLatestFrom(currentTest$.pipe(
+          select(getTestData),
+          select(getEco))),
+        map(([outcome, eco]) =>
+          this.outcomeBehaviourProvider.isVisible(outcome, 'eco', eco)),
+      ),
+      displayEta$: currentTest$.pipe(
+        select(getTestOutcome),
+        withLatestFrom(currentTest$.pipe(
+          select(getTestData),
+          select(getETA))),
+        map(([outcome, eta]) =>
+          this.outcomeBehaviourProvider.isVisible(outcome, 'eta', eta)),
+      ),
+      displayDrivingFault$: currentTest$.pipe(
+        select(getTestOutcome),
+        withLatestFrom(currentTest$.pipe(
+          select(getTestData),
+        )),
+        map(([outcome, data]) =>
+          this.outcomeBehaviourProvider.isVisible(
+            outcome,
+            'faultComment',
+            this.faultSummaryProvider.getDrivingFaultsList(data, TestCategory.BE),
```

Comment:
Needs updating

Reply:
I'll add todos instead. We have had some issues with just choosing the defaults on these functions